tests/testTriggers.go

```diff
@@ -2124,6 +2128,150 @@ func TriggerBackupScaleMongo(contexts *[]*scheduler.Context, recordChan *chan *E
 	})
 }
 
+// TriggerPoolResizeDisk peforms resize-disk on the storage pools for the given contexts
+func TriggerPoolResizeDisk(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: PoolResizeDisk,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+	Step("get storage pools and perform resize-disk by 10 percentage on it ", func() {
+		time.Sleep(1 * time.Minute)
+		for _, ctx := range *contexts {
+			var appVolumes []*volume.Volume
+			var err error
+			Step("Get StoragePool IDs from the app volumes", func() {
+				appVolumes, err = Inst().S.GetVolumes(ctx)
+				UpdateOutcome(event, err)
+				if len(appVolumes) == 0 {
+					UpdateOutcome(event, fmt.Errorf("found no volumes for app %s", ctx.App.Key))
+				}
+				poolSet := make(map[string]struct{})
+				var exists = struct{}{}
+				for _, vol := range appVolumes {
+					replicaSets, err := Inst().V.GetReplicaSets(vol)
+					UpdateOutcome(event, err)
+
+					for _, poolUUID := range replicaSets[0].PoolUuids {
+						logrus.Infof("Pool UUID: %v, Vol: %v", poolUUID, vol.Name)
+						poolSet[poolUUID] = exists
+
+					}
+				}
+
+				for id := range poolSet {
+					err = Inst().V.ResizeStoragePoolByPercentage(id, 2, uint64(10))
```

Comment:
We should make the percentage parameter  such that it can be passed as a torpedo param. That way we can change it in the future w/o having  to touch the code.

Reply:
will be taking this as enhancement for next sprint